awx/api/metadata.py

```diff
@@ -128,7 +128,7 @@ def get_field_info(self, field):
         # Special handling of notification configuration where the required properties
         # are conditional on the type selected.
         if field.field_name == 'notification_configuration':
-            for (notification_type_name, notification_tr_name, notification_type_class) in NotificationTemplate.NOTIFICATION_TYPES:
+            for notification_type_name, notification_tr_name, notification_type_class in NotificationTemplate.NOTIFICATION_TYPES:
```

Comment:
this makes me uncomfortable

Reply:
Never mind, I think I've convinced myself that since this is a `for`, it really is standard practice to not use (). The behavior is the same in either case.